code/controllers/subsystems/processing/projectiles.dm

```diff
@@ -0,0 +1,6 @@
+PROCESSING_SUBSYSTEM_DEF(projectiles)
+	name = "Projectiles"
+	priority = SS_PRIORITY_PROJECTILES
+	flags = SS_TICKER|SS_NO_INIT
```

Comment:
Pls test this as a non-SS_TICKER SS with 1 wait, I'd like to keep tickers to a minimum.

Reply:
I am not entirely sure what you're asking me to do here.